dask_jobqueue/core.py

```diff
@@ -145,12 +171,12 @@ def __init__(self,
         dask_worker_command = (
             '%(python)s -m distributed.cli.dask_worker' % dict(python=sys.executable))
         self._command_template = ' '.join([dask_worker_command, self.scheduler.address])
-        if threads is not None:
-            self._command_template += " --nthreads %d" % threads
+        if self.worker_threads is not None:
+            self._command_template += " --nthreads %d" % self.worker_threads
         if processes is not None:
             self._command_template += " --nprocs %d" % processes
-        if memory is not None:
-            self._command_template += " --memory-limit %s" % memory
+        if self.worker_memory is not None:
+            self._command_template += " --memory-limit %s" % format_bytes(self.worker_memory).replace(' ', '')
```

Comment:
can we strip the space chars in `format_bytes`?

Reply:
Maybe.  This will have to happen in dask/dask regardless.

Reply:
I'm also not sure that we always want to.  Often when rendering bytes for human readers having a space is nicer.  I'm inclined to leave this as an explicit step.